nmap.go

```diff
@@ -56,7 +54,7 @@ func NewScanner(options ...func(*Scanner)) (*Scanner, error) {
 }
 
 // Run runs nmap synchronously and returns the result of the scan.
-func (s *Scanner) Run() (*Run, error) {
+func (s *Scanner) Run() (*Run, []string, error) {
```

Comment:
WDYT of naming the returns here, to make it easier to understand what the function returns by just looking at the prototype?

```suggestion
func (s *Scanner) Run() (result *Run, warnings []string, err error) {
```